bot/exts/utils/unfurl_message_links.py

```diff
@@ -0,0 +1,35 @@
+import discord
+from discord.ext.commands import Cog
+
+from bot.bot import Bot
+from bot.constants import Channels, Guild
+from bot.utils.channel import is_staff_channel
+from bot.utils.messages import extract_message_links, make_message_link_embed
+
+
+class UnfurlMsgLinks(Cog):
+    """Unfurl message links and send embeds containing information about the linked message."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @Cog.listener()
+    async def on_message(self, message: discord.Message) -> None:
+        """Unfurl messages linked in the message if any and send the message link embed."""
+        if (
+            not message.guild
+            or message.guild.id != Guild.id
+            or message.channel.id != Channels.incidents
```

Comment:
this should be `==`

Reply:
Shivansh gave me permissions to push, so its fixed in https://github.com/python-discord/bot/pull/2021/commits/37098dfa7138bbfa7734983496bf7810692d1d1b